src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/profiler/ProfilerEditingTarget.java

```diff
@@ -116,130 +128,122 @@ public String getTabTooltip()
    {
       return new HashSet<AppCommand>();
    }
-   
+
    @Override
    public boolean canCompilePdf()
    {
       return false;
    }
-   
-   
+
    @Override
    public void verifyCppPrerequisites()
    {
    }
-      
+
    @Override
    public Position search(String regex)
    {
       return null;
    }
-   
+
    @Override
    public Position search(Position startPos, String regex)
    {
       return null;
    }
-   
+
    @Override
    public void forceLineHighlighting()
    {
    }
 
-
    public void focus()
    {
    }
-   
+
    public void onActivate()
    {
    }
 
    public void onDeactivate()
-   {      
-      recordCurrentNavigationPosition(); 
+   {
+      recordCurrentNavigationPosition();
    }
 
    @Override
    public void onInitiallyLoaded()
-   {    
+   {
    }
-   
+
    @Override
    public void recordCurrentNavigationPosition()
    {
-      events_.fireEvent(new SourceNavigationEvent(
-            SourceNavigation.create(
-            getId(), 
-            getPath(), 
-            SourcePosition.create(0, 0))));
+      events_.fireEvent(new SourceNavigationEvent(SourceNavigation
+            .create(getId(), getPath(), SourcePosition.create(0, 0))));
    }
-   
+
    @Override
-   public void navigateToPosition(SourcePosition position, 
+   public void navigateToPosition(SourcePosition position,
                                   boolean recordCurrent)
-   {   
+   {
    }
-   
-   
+
    @Override
-   public void navigateToPosition(SourcePosition position, 
-                                  boolean recordCurrent,
-                                  boolean highlightLine)
-   {   
+   public void navigateToPosition(SourcePosition position,
```

Comment:
This looks like an Eclipse code reformat. We should also avoid this sort of change (except in commits dedicated to reformatting or whitespace) to avoid the noisy diff issue Kevin mentioned)